### PR TITLE
Added recipe for BiorbdOptim

### PR DESCRIPTION
**Pull request by pariterre**

### CODE REVIEW

**Comment by pariterre** [comment]
@conda-forge/staged-recipes 
I think this is ready to merge :)

**Comment by pariterre** [comment]
@conda-forge/staged-recipes
I think this is ready to merge :)

File: recipes/biorbd-optim/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "biorbd-optim" %}
+{% set version = "0.0.1" %}
+{% set build_num = 1 %}
```
Comment (scopatz) [inline recipes/biorbd-optim/meta.yaml:3]: Please remove

File: recipes/biorbd-optim/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pyomeca/BiorbdOptim/archive/Release_{{ version }}.tar.gz
+  sha256: 3000cfc8469293685a8c2c71ab2cb401c2eab7b6fbd12089f83b7933d772e959
+  
+build:
+  number: {{ build_num }}
```
Comment (scopatz) [inline recipes/biorbd-optim/meta.yaml:14]: Build number should be zero

File: recipes/biorbd-optim/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+  string: py{{ py }}_h{{ PKG_HASH }}_{{ build_num }}
```
Comment (scopatz) [inline recipes/biorbd-optim/meta.yaml:17]: Please remove

File: recipes/biorbd-optim/yum_requirements.txt [hunk 4]
```diff
@@ -0,0 +1,2 @@
+xorg-x11-server-Xvfb
+libXtst
```
Comment (scopatz) [inline recipes/biorbd-optim/yum_requirements.txt:2]: These should have CDTs associated with them

Comment (pariterre) [reply recipes/biorbd-optim/yum_requirements.txt:2]: Thanks for the review!
I don't know what you mean, do you have an example that I can use?

Comment (scopatz) [reply recipes/biorbd-optim/yum_requirements.txt:2]: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#host

**Comment by pariterre** [comment]
@scopatz 
Requested changes done! :)

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**